test/org/sagebionetworks/bridge/play/controllers/ReportControllerTest.java

```diff
@@ -410,8 +415,76 @@ public void adminCanDeleteParticipantIndex() {
     }
     
     @Test(expected = UnauthorizedException.class)
-    public void nonAdminCannotDeleteParticipantIndex() {
-        controller.deleteParticipantReportIndex(REPORT_ID);
+    public void nonAdminCannotDeleteParticipantIndex() throws Exception {
+        Result result = controller.deleteParticipantReportIndex(REPORT_ID);
+        assertResult(result);
```

Comment:
Will this assert ever fire? From a reading of the code and the test, it looks like the delete() call will always through UnauthorizedException.